### ISSUE TITLE
app crashes when using back button

### ISSUE THREAD

**Issue by pedrorijo91** [pedrorijo91/garmin-padel#72]
device: Fenix 7 Pro Sapphire Solar 
OS 14.36
app version: 1.6.1

see video 

https://github.com/pedrorijo91/garmin-padel/assets/1999050/2f3f85de-d743-4339-99f3-d749ed1a4b89

**Comment by pedrorijo91** [comment]
simulator shows a different UI and behaviour 🤔 

<img width="428" alt="image" src="https://github.com/pedrorijo91/garmin-padel/assets/1999050/686fc5dc-c9cc-40ff-bbce-e48a6ccd52c6">

**Comment by pedrorijo91** [comment]
forum question raised at https://forums.garmin.com/developer/connect-iq/f/discussion/348873/different-behaviour-and-ui-between-simulator-and-real-watch

**Comment by pedrorijo91** [comment]
able to reproduce in a real device

**Comment by pedrorijo91** [comment]
just tested, still happening. let's wait for emulator to be updated to make it easier to reproduce and test fixes  🤷

**Comment by pedrorijo91** [comment]
thanks @wschoot it's kinda silly, because that view is a 'confirmationDelegate', native from garmin SDK (see https://github.com/pedrorijo91/garmin-padel/blob/master/source/delegates/UndoConfirmationDelegate.mc#L5)

so in theory it should be covered. a possible solution is to add a handler to more keys. but then i can't test it because the simulator displays a different UI 🥲

<img width="435" alt="Image" src="https://github.com/user-attachments/assets/a15cc2b3-783e-4c47-bc1c-be0b930d3214" />

**Comment by wschoot** [comment]
Yep, I read about differences between simulator and real world. Can I test on my watch for you and show you how it works somehow, if thats helpfull??
